src/main/resources/assets/js/components/DeploymentComponent.jsx

```diff
@@ -23,8 +23,10 @@ define([
       this.setState({loading: bool});
     },
 
-    handleDestroyDeployment: function () {
-      this.props.destroyDeployment(this.props.model, this);
+    handleDestroyDeployment: function(forceStop) {
+      return function() {
+        this.props.destroyDeployment(this.props.model, {forceStop: forceStop}, this);
+      }.bind(this);
```

Comment:
I did this, because I don't want 'repeat myself' with this line in 2 similiar handler functions:
this.props.destroyDeployment(this.props.model, {forceStop: forceStop}, this);

Otherwise I have to define 2 clickhandler functions like this:
```
handleDestroyDeployment: function() {
  this.props.destroyDeployment(this.props.model, {forceStop: false}, this);
},
handleDestroyDeploymentWithForceStop: function() {
  this.props.destroyDeployment(this.props.model, {forceStop: true}, this);
}
```
A lot of similiar code..